Kitodo/src/main/java/de/sub/goobi/export/download/ExportMets.java

```diff
@@ -150,8 +150,13 @@ protected boolean writeMetsFile(Process process, URI metaFile, Fileformat gdzfil
         MetsModsImportExport mm = new MetsModsImportExport(this.myPrefs);
         mm.setWriteLocal(writeLocalFileGroup);
         mm = serviceManager.getSchemaService().tempConvert(gdzfile, this, mm, this.myPrefs, process);
-        mm.write(metaFile.toString());
-        Helper.setMeldung(null, process.getTitle() + ": ", "ExportFinished");
-        return true;
+        User user = (User) Helper.getManagedBeanValue("#{LoginForm.myBenutzer}");
```

Comment:
This should go into `ProzessverwaltungForm` class and there into the methods `exportMets`, `exportPdf` and `exportDMS`.